cmd/export.go

```diff
@@ -39,6 +41,18 @@ func runExport(cmd *cobra.Command, args []string) {
 		OutputFolder: outputDir,
 		MetadataOnly: metadataOnly,
 	}
-
 	entityDumper.DumpChannelData(options)
+	var versionfile string
+	versionfile = options.GetOutputFolderAbsPath() + "/version.txt"
+	vf, err := os.OpenFile(versionfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	defer vf.Close()
+	if os.IsNotExist(err) {
+		f, err := os.Create(versionfile)
+		if err != nil {
+			log.Fatal().Msg("Unable to create version file")
+		}
+		vf = f
+	}
+
+	vf.WriteString(utils.GetCurrentServerVersion() + "\n")
```

Comment:
should write in the style of `version=0.0.0` since we may want to add more properties to here